packer/ansible/roles/mongo/tasks/main.yml

```diff
@@ -33,7 +33,7 @@
 - name: Remove --quiet mongo command line option
   lineinfile:
     dest: /lib/systemd/system/mongod.service
-    regexp: ^ExecStart=/usr/bin/mongod --quiet --config /etc/mongod.conf
+    regexp: ^ExecStart=/usr/bin/mongod --quiet
```

Comment:
I mean, it could even be shorter.  
^ExecStart=  
There is only one ExecStart line I'm guessing, and you want it.